app/helpers/application_helper.rb

```diff
@@ -28,10 +28,14 @@ def policy_service_name(policy = nil)
   end
 
   def feedback_url
-    "https://docs.google.com/forms/d/e/1FAIpQLSdAyOxHme39E8lMnD2qY029mmk4Lpn84soYg2vLrT5BV9IUSg/viewform?usp=sf_link"
+    current_policy&.feedback_url
   end
 
   def start_page_url
-    Policies[current_policy_routing_name].start_page_url
+    current_policy&.start_page_url
```

Comment:
Is there a reason why the safe navigation operator has snuck in here? I feel like the service is currently set up such that anywhere we use these helpers to link to the start page or feedback url we are in the context of a policy. Generally if we can avoid a method returning `nil` then we should as it is one of the causes of strange and subtle bugs. For example, say a page _was_ introduced where there is no `current_policy`. Would it be better for these methods to return `nil`, and therefor create links to nowhere, or would it be better for a spec for that page to blow up so that we notice the missing dependency?

Reply:
Good thinking, I um-ed and ah-ed about this too, but yeah, I think blowing up would be the preferred behaviour, so we could fix it one way or the other.